content/en/docs/contributing/_index.md

```diff
@@ -36,13 +37,14 @@ To get invites you can subscribe to [our mailing list](https://groups.google.com
 We have 2 regular repeating meetings:
 
 * our daily stand-up meetings [on Google Meet](
-https://meet.google.com/eum-fyvt-xpa) at 10:30 London time every weekday 
-* our bi-weekly developer meetings [are on Google Meet](https://meet.google.com/abp-bwhk-wxc) 17:00 London time every 2 weeks (check calendar invites or [meeting notes](https://docs.google.com/document/d/1Tc5t6ylY9dhXAan1OjOoldeaoys1Yh4Ir710ATfBa5U)). 
+https://meet.google.com/eum-fyvt-xpa) at 10:30 London time every weekday
+* our bi-weekly developer meetings [are on Google Meet](https://meet.google.com/abp-bwhk-wxc) 17:00 London time every 2 weeks (check calendar invites or [meeting notes](https://docs.google.com/document/d/1Tc5t6ylY9dhXAan1OjOoldeaoys1Yh4Ir710ATfBa5U)).
 
 You can check what time this is where you are using [`everytimezone.com`](https://everytimezone.com/).
 
 ## Slack
+
 We have two cert-manager channels on [the Kubernetes Slack](https://slack.k8s.io) which we use to communicate:
 
-* `cert-manager` a general users channel, use this one for any usage related questions.
-* `cert-manager-dev` this is what the developers use to collaborate, please only use this for code related questions
+* `cert-manager`: a general channel for all users of cert-manager; use this one for any usage related questions.
+* `cert-manager-dev`: a channel for collaboration between cert-manager developers and maintainers; please only use this for code related questions
```

Comment:
some people come to help on the documentation and might not be developers per-se 😅

```suggestion
* `cert-manager-dev`: a channel for collaboration between cert-manager contributors and maintainers; please only use this for code related questions
```

Reply:
ah yeah, excellent point, I'll make the change!